wolox-books/src/app/services/shopping-cart.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ShoppingCartService {
+
+  private count = new BehaviorSubject(0);
+  bookCounter = this.count.asObservable();
+
+  constructor() { }
+
+  addBook(bookCount) {
```

Comment:
To create a service only for a count feels incomplete. Maybe you can receive a book in this method and add it to an array? That way you will know which book you have in the cart and of course the count 😄